services/movie_session.py

```diff
@@ -42,3 +42,11 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
+
+
+def get_taken_seats(movie_session_id: int) -> list[dict]:
+    session = get_movie_session_by_id(movie_session_id)
+    return [
```

Comment:
You can also use values() method here which will pull fields that you need : `list(session.tickets.all().values('row', 'seat'))`